glide.yaml

```diff
@@ -15,6 +15,7 @@ import:
   version: ^1
 - package: go.uber.org/thriftrw
   version: ^1
+- package: github.com/gogo/protobuf
```

Comment:
I'll see heh. If CoreOS, Docker, and Kubernete's can't though, we probably won't have much luck